platforms/chibios/pin_defs.h

```diff
@@ -53,6 +53,58 @@
 #        define D5 PAL_LINE(GPIOC, 13)
 #        define B0 PAL_LINE(GPIOC, 14)
 #    endif
+
+#elif defined(CONVERT_TO_STEMCELL)
```

Comment:
Just a note for us in general.... we really need to work out a better solution for these, instead of dumping them all into a single file.